### ISSUE TITLE
Creating a termbase

### ISSUE THREAD

**Issue by amatefi** [paulfilkin/tradosClarity#14]
- Language Fields Window: On the section “Select language” the fields are not labeled so the screen reader will just read Edit.
- General Termbase Window: To navigate through the table you need to use the navigation table. On the column Languages, the languages of the termbase are not read
- no feedback from screen reader after using Next button to navigate from one window to another
- The import history dropdown list is inaccessible, and the same issue affects the export dropdown list
- Add new terms in the Termbase manually --> click on the Termbase name by pressing Enter twice. Search for New entry button and press it. Please note that the term addition process is currently not accessible.
